.github/workflows/automation-benchmark-tests.yml

```diff
@@ -20,17 +20,22 @@ on:
         required: true
         default: benchmark
         type: string
+      team:
+        description: Team to run the tests for (e.g. BIX, CCIP)
+        required: true
+        type: string
 
 jobs:
   run-e2e-tests-workflow:
     name: Run E2E Tests
-    uses: smartcontractkit/.github/.github/workflows/run-e2e-tests.yml@5412507526722a7b1c5d719fa686eed5a1bc4035 # ctf-run-tests@0.2.0
+    uses: smartcontractkit/.github/.github/workflows/run-e2e-tests.yml@0d4a2b2b009c87b5c366d0b97f7a8d7de2f60760
```

Comment:
Linter will probably yell at you for not including a version comment

Reply:
there is no version for reusable workflows... :/